test/runtests.jl

```diff
@@ -1,14 +1,17 @@
 using Cclib
 using PythonCall
 using Test
+using AtomsBase
+
 
 
 @testset "Cclib.jl" begin
+    test_file = "./data/uracil_two.xyz"
 
     # Check that input is read correctly
-    @test ccread("./data/uracil_two.xyz")["natom"] == 12
-    @test ccread("./data/uracil_two.xyz")["atomnos"] == [7, 6, 1, 8, 7, 6, 8, 6, 6, 1, 1, 1]
-    @test ccread("./data/uracil_two.xyz")["atomcoords"] == [5.435 4.091 6.109 3.647 3.253 3.636 2.79 5.065 5.906 2.243 5.448 7.003;
+    @test ccread(test_file)["natom"] == 12
+    @test ccread(test_file)["atomnos"] == [7, 6, 1, 8, 7, 6, 8, 6, 6, 1, 1, 1]
+    @test ccread(test_file)["atomcoords"] == [5.435 4.091 6.109 3.647 3.253 3.636 2.79 5.065 5.906 2.243 5.448 7.003;
```

Comment:
I think eric was saying can we do
```
    test_data = ccread(test_file)
    @test test_data["natom"] == 12
    @test test_data["atomnos"] == [7, 6, 1, 8, 7, 6, 8, 6, 6, 1, 1, 1]
    @test test_data["atomcoords"] == [5.435 4.091 6.109 3.647 3.253 3.636 2.79 5.065 5.906 2.243 5.448 7.003;
```

This prevents reading the file multiple times

Reply:
done